rust_src/src/dired.rs

```diff
@@ -0,0 +1,54 @@
+//! Lisp functions for making directory listings.
+
+#[cfg(not(windows))]
+use libc;
+
+use remacs_macros::lisp_fn;
+#[cfg(not(windows))]
+use remacs_sys::build_string;
+use remacs_sys::globals;
+
+use lisp::{defsubr, LispObject};
+#[cfg(not(windows))]
+use lists::list;
+
+fn get_user_real_login_name() -> LispObject {
+    unsafe { globals.f_Vuser_real_login_name }
+}
+
+#[cfg(windows)]
+fn get_users() -> LispObject {
```

Comment:
Should this not return a one element list to act like the other `get_users`?

Reply:
Yes it should -- good catch -- I just pushed a fix and added a test to catch it and similar problems.